extensions/moji/src/index.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+  randomId,
+} from "@raycast/api";
+import { useState, useEffect, useRef } from "react";
+import fetch, { AbortError } from "node-fetch";
+
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List isLoading={state.isLoading} onSearchTextChange={search} searchBarPlaceholder="Search..." throttle>
+      <List.Section title="Results" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult) => (
+          <SearchListItem key={searchResult.id} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      title={searchResult.title}
+      accessoryTitle={searchResult.name}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <OpenInBrowserAction title="Open in Browser" url={searchResult.url} />
+          </ActionPanel.Section>
+          <ActionPanel.Section>
+            <CopyToClipboardAction
+              title="Copy"
+              content={searchResult.title}
+              shortcut={{ modifiers: ["cmd"], key: "." }}
+            />
+          </ActionPanel.Section>
+        </ActionPanel>
+      }
+    />
+  );
+}
+
+function useSearch() {
+  const [state, setState] = useState<SearchState>({ results: [], isLoading: true });
+  const cancelRef = useRef<AbortController | null>(null);
+
+  useEffect(() => {
+    search("");
+    return () => {
+      cancelRef.current?.abort();
+    };
+  }, []);
+
+  async function search(searchText: string) {
+    cancelRef.current?.abort();
+    cancelRef.current = new AbortController();
+    try {
+      setState((oldState) => ({
+        ...oldState,
+        isLoading: true,
+      }));
+      const results = await performSearch(searchText, cancelRef.current.signal);
+      setState((oldState) => ({
+        ...oldState,
+        results: results,
+        isLoading: false,
+      }));
+    } catch (error) {
+      if (error instanceof AbortError) {
+        return;
+      }
+      console.error("search error", error);
+      showToast(ToastStyle.Failure, "Could not perform search", String(error));
+    }
+  }
+
+  return {
+    state: state,
+    search: search,
+  };
+}
+
+async function performSearch(searchText: string, signal: AbortSignal): Promise<SearchResult[]> {
+  let response = await fetch("https://api.mojidict.com/parse/functions/search_v3", {
+    method: "POST",
+    signal: signal,
+    body: JSON.stringify({
+      searchText: searchText,
+      langEnv: "zh-CN_ja",
+      _InstallationId: "7d959a18-48c4-243c-7486-632147466544",
+      _ClientVersion: "js2.12.0",
+      _ApplicationId: "E62VyFVLMiW7kvbtVq3p",
```

Comment:
Is this your credentials?

Reply:
These are fixed parameters, not related to the user.
Check [this](https://github.com/search?q=E62VyFVLMiW7kvbtVq3p&type=code).